Update de.html l. 1001-1250

Odakova2 opened the pull request.

Odakova2 committed:
Update de.html l. 1001-1250

herrhundt commented on de.html:
```diff
@@ -1174,7 +1174,7 @@
 <panel w=450 h=90>
 <words w=450 x=-15 no-bg>
-  Aber die <i>wichtigste</i> Verknüpfung von allen ist, das Gelernte an etwas mit Bedeutung zu knüpfen.
+  Aber die <i>wichtigste</i> Verknüpfung von allen ist, dass Gelernte an etwas mit Bedeutung zu knüpfen.
```
```suggestion
  Aber die <i>wichtigste</i> Verknüpfung von allen ist, das Gelernte an etwas mit Bedeutung zu verbinden.
```

its-nico replied:
```suggestion
  Aber die <i>wichtigste</i> Verknüpfung von allen ist, das Gelernte mit einer Bedeutung zu verknüpfen.
```

herrhundt commented on de.html:
```diff
@@ -1234,7 +1234,7 @@
 <pic src="pics/leit2.png" sx=450 sy=950></pic>
 <words x=10 y=10 w=400 h=60>
-	Ich glaube, das ist der beste Weg, um beim Lernen motiviert zu bleiben:
+	Ich glaube, dass ist der beste Weg, um beim Lernen motiviert zu bleiben:
```
```suggestion
	Ich glaube, das ist der beste Weg, um beim Lernen motiviert zu bleiben:
```

its-nico committed:
Update de.html

Co-authored-by: herrhundt <59929867+herrhundt@users.noreply.github.com>

its-nico committed:
Update de.html

Hinweis zu L 1005: isoliert macht keinen Sinn und ist falsch aus dem Englischen übersetzt, dort wurde von "disconnected" gesprochen.

noahdyj committed:
Update de.html

Co-authored-by: its-nico <65342313+its-nico@users.noreply.github.com>
